From ead80e80c41a4c1275543c3c7c2d22ff1c19717a Mon Sep 17 00:00:00 2001
From: Myy <myy@miouyouyou.fr>
Date: Mon, 21 Aug 2017 06:22:15 +0000
Subject: [PATCH] net: stmmac: Reverting a part of "Use the right logging
 function"

Turns out that the phy_attached_info call, added in commit
fbca164776e438b639af592c522b8b0506b54dcc provokes a crash on boot
with MiQi devices.

Signed-off-by: Myy <myy@miouyouyou.fr>
---
 drivers/net/ethernet/stmicro/stmmac/stmmac_mdio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/stmicro/stmmac/stmmac_mdio.c b/drivers/net/ethernet/stmicro/stmmac/stmmac_mdio.c
index 72ec711f..889810f0 100644
--- a/drivers/net/ethernet/stmicro/stmmac/stmmac_mdio.c
+++ b/drivers/net/ethernet/stmicro/stmmac/stmmac_mdio.c
@@ -286,7 +286,7 @@ int stmmac_mdio_register(struct net_device *ndev)
 			irq_str = irq_num;
 			break;
 		}
-		phy_attached_info(phydev);
+		//phy_attached_info(phydev);
 		found = 1;
 	}
 
-- 
2.13.0

